gen/models/resources/api-gateway-method.json

```diff
@@ -0,0 +1,71 @@
+{
+  "Name": "ApiGatewayVerb",
```

Comment:
Hmm, I knew the day would come that we would see a prefix conflict. I'll have to add a new optional field in these templates called `lensPrefix` so we can give this a proper name and have the prefix be something like `agme`. No need to change anything, I'll make a note and fix it.